File: BE-Movie/src/main/java/com/a2m/service/CloudinaryService.java
package com.a2m.service;

import org.springframework.web.multipart.MultipartFile;

import java.io.IOException;
import java.util.List;

public interface CloudinaryService {
    List<String> upload(MultipartFile[] files) throws IOException;

    void delete(String publicId) throws IOException;
}

File: FE/src/app/services/cloudinary/cloudinary.service.ts
import { Injectable } from '@angular/core';
import { CloudinaryApiService } from './cloudinary-api.service';

@Injectable({
  providedIn: 'root'
})
export class CloudinaryService {

constructor(private readonly uploadService: CloudinaryApiService) { }

  upload(files: any) {
    return this.uploadService.upload(files);
  }

  delete(publicId: any) {
    return this.uploadService.delete(publicId);
  }

}
